src/util/utilization.jl

```diff
@@ -0,0 +1,126 @@
+
+"""
+    `run_UTIL(data::Dict{String,<:Any})`
+
+Returns a priority restoration order for damaged components in a single network.
+
+# Example
+```
+julia> run_UTIL(data)
+Dict{String, Any} with 6 entries:
+    "4" => [("branch", "2")]
+    "1" => [("gen", "1")]
+    "5" => [("branch", "3")]
+    "2" => [("gen", "2")]
+    "6" => [("gen", "3")]
+    "3" => [("branch", "1")]
+```
+"""
+function run_UTIL(data::Dict{String,<:Any})
+    if _IM.ismultinetwork(data)
+        Memento.error(_PM._LOGGER, "run_UTIL requires a single network.")
+    end
+
+    d_comp_vec = vcat([[(comp_type,comp_id) for comp_id in comp_ids] for (comp_type,comp_ids) in get_repairable_components(data)]...)
```

Comment:
`vcat` is not used in PowerModels, so I would suggest to use a for loop or a list comprehension instead.

Reply:
changed to a for loop